src/aat/resources/features/F-047/F-047.feature

```diff
@@ -0,0 +1,108 @@
+
+  @S-098
+  Scenario: must return 200 and an empty list if no case is found
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an userId which doesn't have access to the case]
+    And it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [contains an empty list of case ids, along with an HTTP-200 OK]
+    And the response has all other details as expected
+
+  @S-099 @Ignore # Response code mismatch, expected: 401, actual: 403 Will be fixed as a part of RDM-6628
+  Scenario: must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide valid authentication credentials]
+    And it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [contains an HTTP-401 Unauthorized]
+    And the response has all other details as expected
+
+  @S-100
+  Scenario: must return 403 when request provides authentic credentials without authorised access to the operation
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide authorised access to the operation]
+    And it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [contains an HTTP-403 Forbidden]
+    And the response has all other details as expected
+
+  @S-570 @Ignore # Response code mismatch, expected: 400, actual: 500
+  Scenario: must return negative response HTTP-400 when request contains a malformed user ID
+    Given a user with [an inactive profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a malformed user ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-571 @Ignore # Response code mismatch, expected: 400, actual: 500
+  Scenario: must return negative response HTTP-400 when request contains a malformed case type ID
+    Given a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a malformed case type ID]
+    And   it is submitted to call the [Get case ids] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-572 @Ignore # Response code mismatch, expected: 400, actual: 403
+  Scenario: must return negative response HTTP-400 when request contains a malformed jurisdiction ID
```

Comment:
Can you please add defect number?
Also add this scenario in RDM-6387.

Reply:
done